tests/allocator/allocator.cpp

```diff
@@ -60,7 +60,7 @@ struct foo {
 	{
 		bar = 1;
 		for (int i = 0; i < TEST_ARR_SIZE; ++i)
-			arr[i] = i;
+			arr[i] = static_cast<char>(i);
```

Comment:
if this were a real code, I would have suggested you change the array type to int, but in test, w/e...